web_src/js/index.js

```diff
@@ -2628,9 +2627,11 @@ $(document).ready(async () => {
     }
   });
 
+  const codeBlocks = document.querySelectorAll('pre code');
+
   // parallel init of lazy-loaded features
-  [hljs] = await Promise.all([
-    initHighlight(),
+  await Promise.all([
+    (codeBlocks && codeBlocks.length) && highlight(codeBlocks),
```

Comment:
Double check (here and in `highlight`) seems unnecessary?

Reply:
Indeed. Guess I was getting a bit overzealous with those safeguards.